res/controllers/Pioneer-DDJ-400.midi.xml

```diff
@@ -0,0 +1,4877 @@
+            </control>
+            <control>
+                <description>LOOP IN/4 BEAT LONG PRESS (DECK1) - press - Enable Auto 4 Beat loop</description>
+                <group>[Channel1]</group>
+                <key>PioneerDDJ400.loopin4beatPressedLong</key>
+                <status>0x90</status>
+                <midino>0x14</midino>
+                <options>
+                    <Script-Binding/>
+                </options>
+            </control>
+
+            <control>
+                <description>LOOP IN/4 BEAT (DECK2) - press - (loop off) set Loop point, (loop on) adjust loop in point
+                    with jog wheel
+                </description>
+                <group>[Channel2]</group>
+                <key>PioneerDDJ400.loopin4beatPressed</key>
+                <status>0x91</status>
+                <midino>0x10</midino>
+                <options>
+                    <Script-Binding/>
+                </options>
+            </control>
+            <control>
+                <description>LOOP IN/4 BEAT LONG PRESS (DECK2) - press - Enable Auto 4 Beat loop</description>
+                <group>[Channel2]</group>
+                <key>PioneerDDJ400.loopin4beatPressedLong</key>
+                <status>0x91</status>
+                <midino>0x14</midino>
+                <options>
+                    <Script-Binding/>
+                </options>
+            </control>
+
+            <control>
+                <description>LOOP OUT (DECK1) - press - (loop off) set Loop out point, (loop on) adjust loop out point
+                    with jog wheel
+                </description>
+                <group>[Channel1]</group>
+                <key>PioneerDDJ400.loopoutPressed</key>
+                <status>0x90</status>
+                <midino>0x11</midino>
+                <options>
+                    <Script-Binding/>
+                </options>
+            </control>
+            <control>
+                <description>LOOP OUT (DECK2) - press - (loop off) set Loop out point, (loop on) adjust loop out point
+                    with jog wheel
+                </description>
+                <group>[Channel2]</group>
+                <key>PioneerDDJ400.loopoutPressed</key>
+                <status>0x91</status>
+                <midino>0x11</midino>
+                <options>
+                    <Script-Binding/>
+                </options>
+            </control>
+
+            <control>
+                <description>RELOOP/EXIT (DECK1) - press - (loop off) Reloop, (loop on) Loop exit</description>
+                <group>[Channel1]</group>
+                <key>reloop_toggle</key> <!-- check if correct -->
+                <status>0x90</status>
+                <midino>0x4D</midino>
+                <options>
+                    <Normal/>
+                </options>
+            </control>
+            <control>
+                <description>RELOOP/EXIT +SHIFT (DECK1) - press - Active loop on/off</description>
```

Comment:
What this does:
> Activate current loop, jump to its loop in point, and stop playback.

So that description is a bit misleading. Also, the wiki entry has identical descriptions for [RELOOP/EXIT] and [SHIFT]+[RELOOP/EXIT].

Reply:
Ping.

Reply:
Fixed both the description and the wiki.